tests/integration/backend.py

```diff
@@ -44,6 +49,7 @@ def __init__(
         instrument: str = 'dummy',
         dev: bool = True,
         log_level: int | str = logging.INFO,
+        transport: Literal['kafka', 'none'] = 'kafka',
```

Comment:
Consider using
```suggestion
        transport: Literal['kafka', None] = 'kafka',
```
?

Reply:
This is actually deliberate and good I think: `None` is often interpreted as "use default", but here `'none'` refers to a particular kind of transport.

Reply:
Then how about calling it `'null'`?